sonic_platform_base/sonic_sfp/sfputilbase.py

```diff
@@ -510,12 +511,19 @@ def read_porttab_mappings(self, porttabfile, asic_inst=0):
                 elif "asic_port_name" not in title and len(line.split()) >= 4:
                     fp_port_index = int(line.split()[3])
                 else:
-                    fp_port_index = portname.split("Ethernet").pop()
+                    front_panel_prefix = re.match(FRONT_PANEL_PORT_REGEX, portname)
+                    if not front_panel_prefix:
+                        continue
+                    front_panel_prefix = front_panel_prefix.group(1)
```

Comment:
Could you not to reuse the same variable name `front_panel_prefix` for different objects?

Reply:
sure